core/common/assets/js/api/core/backwards-compatibility/commands.js

```diff
@@ -0,0 +1,26 @@
+export default class CommandsBackwardsCompatibility extends elementorModules.Module {
+	__construct() {
+		this.onOrig = this.on;
+	}
+
+	on = ( eventName, callback ) => {
```

Comment:
```suggestion
	on( eventName, callback ) {
```
Why it's in a different syntax?

Reply:
Because it extends a non ES6 object, it doesn't work.

Reply:
It doesn't make sense. A lot of our modules extend exactly this none-ES6 module, and they work correctly.

Reply:
@kobizz, you have better solution?

Reply:
@kobizz, we already tried, it doesn't work.
Do you want to check it yourself?  